manifest.json

```diff
@@ -6,7 +6,7 @@
 		"en": "Framapad clone, a online editor providing collaborative editing in real-time.",
 		"fr": "Clone de Framapad, un éditeur en ligne fournissant l'édition collaborative en temps réel."
 	},
-	"version": "1.6.3~ynh2",
+	"version": "1.6.4~ynh2",
```

Comment:
Hello,

I think, while we update the upstream version, the package version should be reset to 1.
For example : `1.6.3~ynh1`, `1.6.3~ynh2`, `1.6.4~ynh1`, `1.6.4~ynh2`, ...

Reply:
I don't know about that, actually I started with this method. And finally I thought it would be better to increment this value for each iteration.
It could be interesting to discuss about that.

Reply:
Yes,
Maybe here it's not the better place to discuss about that but yes it could be good to have the point of view from some packager and after to decide one time.